### DISCUSSION TITLE
RCON settings + extended save

### DISCUSSION THREAD

**Discussion by Darojax** [soda3x/ArmaReforgerServerTool#59] [Ideas]
Thanks for a very nice tool. Would be great if the tool supports settings for RCON, such as is present in the server.json example on the bohemia page:

```
	"rcon": {
		"address": "192.168.9.10",
		"port": 19999,
		"password": "changeme_withoutspaces",
		"permission": "monitor",
		"blacklist": [],
		"whitelist": []
	},
```

It would also be great if when saving the configuration the data on the "Server Management" tab also got saved so one doesnt have to manually fill that in when the tool starts.

**Comment by soda3x** [comment]
its in the pipeline :)
